2016-01-06/swagger/bucket_replication.json

```diff
@@ -0,0 +1,93 @@
+{
+    "parameters": [
+      {
+        "$ref": "#/parameters/bucketName"
+      },
+      {
+        "$ref": "#/parameters/zone"
+      }
+    ],
+    "put": {
+      "tags": [
+        "Bucket"
+      ],
+      "operationId": "PutBucketReplication",
+      "summary": "PUT Bucket Replication",
+      "description": "Set Replication information of the bucket.",
+      "externalDocs": {
+        "url": "https://docs.qingcloud.com/qingstor/api/bucket/replication/put_replication.html"
+      },
+      "parameters": [
+        {
+          "name": "Body",
+          "description": "The request body",
+          "in": "body",
+          "schema": {
+            "type": "object",
+            "properties": {
+              "rules": {
+                "description": "Bucket Replication rules",
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/rules"
+                }
+              }
+            },
+            "required": [
+              "rules"
+            ]
+          }
+        }
+      ],
+      "responses": {
+        "200": {
```

Comment:
`PUT` request returns a 200 response? Please check it again, docs looks wrong:

![image](https://user-images.githubusercontent.com/5351546/98903950-91668500-24f3-11eb-836b-2c5aa3192a33.png)

rest looks good to me.